Code/Framework/AzFramework/AzFramework/Quality/QualityCVarGroup.cpp

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzFramework/Quality/QualityCVarGroup.h>
+#include <AzFramework/Quality/QualitySystemBus.h>
+
+#include <AzCore/Settings/SettingsRegistryImpl.h>

```

Comment:
This can be use the interface SettingsRegistry header instead of the Impl header as this file is not creating a SettingsRegistry instance.
```suggestion
#include <AzCore/Settings/SettingsRegistry.h>

```